translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionDateTimeLikeAssertion.kt

```diff
@@ -11,5 +11,6 @@ enum class DescriptionDateTimeLikeAssertion(override val value: String) : String
     YEAR("Jahr"),
     MONTH("Monat"),
     DAY_OF_WEEK("Wochentag"),
-    DAY("Tag")
+    DAY("Tag"),
+    IS_AFTER_OR_EQUALS("ist am oder nach dem"),
```

Comment:
I realised the text I proposed only works well for ChronoLocalDate, please change to:
```suggestion
    IS_AFTER_OR_EQUALS("ist nach oder zeitgleich"),
```